src/main/java/com/artipie/asto/test/TestResource.java

```diff
@@ -47,12 +48,19 @@
      */
     private final String name;
 
+    /**
+     * The Vert.x file system.
+     */
+    private final FileSystem fls;
+
     /**
      * Ctor.
      * @param name Resource path
+     * @param fls The file system.
      */
-    public TestResource(final String name) {
+    public TestResource(final String name, final FileSystem fls) {
```

Comment:
@Sammers21 that's really not nice to add another argument to this class, as it used in many tests. Adding another argument here will make it way harder to use

Reply:
@olegmoz, what do you suggest?

Reply:
@Sammers21 I suggest to create separate class `VertxFileStorage` and keep `FileStorage` intact. We can switch to using it in Artipie without breaking other stuff